apps/playground/src/app/quiz/components/question/question.component.scss

```diff
@@ -0,0 +1,99 @@
+$font-stack: Space Mono, monospace;
+$font-weight-max: 900;
+
+ol {
+  margin-top: 15px;
+  margin-left: -40px;
+  cursor: pointer;
+}
+ol li {
+  margin-left: 30px;
+}
+
+.radio-options {
+  margin-bottom: 5px;
+  margin-left: 0.5rem;
+  padding: 4px;
+}
+
+.option {
+  border: 2px solid #979797;
+  font-family: $font-stack;
+  font-size: 20px;
+  color: #0f0900;
+  background-color: #f5f5f5;
+  width: 39rem !important;
+  height: auto;
+  padding: 5px 5px 0 30px;
+  margin-left: -5px;
+  vertical-align: middle;
+}
+.option:hover {
+  outline: 2px solid #007aff;
+}
+
+.is-correct {
+  background-color: #00c853 !important;
+}
+.is-incorrect {
+  background-color: #ff0000 !important;
+}
+
+.feedback-icon {
+  position: absolute;
+  right: 0;
+  margin-right: 40px;
```

Comment:
If you're using position absolute, just specify appropriate left/top instead of margins.
Or would be better if you could avoid `absolute`

Reply:
ok I removed position absolute and using margins instead.

Reply:
reverting back to my original CSS because I can't get it to align the way I want otherwise